src/main/java/nerdhub/textilelib/events/render/ScreenDrawEvent.java

```diff
@@ -1,26 +1,25 @@
 package nerdhub.textilelib.events.render;
 
 import nerdhub.textilelib.events.Event;
-import net.fabricmc.api.EnvType;
-import net.fabricmc.api.Environment;
-import net.minecraft.client.gui.Gui;
+import net.fabricmc.api.*;
```

Comment:
We need to come to a consensus on what style we want as these import changes do nothing but increase review.

Reply:
agreed, so what do we prefer?
how many imports qualify a `.*` import, or do we ALWAYS want single imports?